### PR TITLE
Ref/sc 5520

### PR DESCRIPTION
**Pull request by yuliashikhareva**
# Description

### CODE REVIEW

File: runtime/helpers/dateTimeHelpers.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+		* @returns {Date}
+		*
+		* Example use
+		*
+		* stringToDate("17/9/2014","dd/MM/yyyy","/");
+		* stringToDate("9/17/2014","mm/dd/yyyy","/")
+		* stringToDate("9-17-2014","mm-dd-yyyy","-")
+		*/
+        stringToDate: function (_date, _format, _delimiter) {
+            let formatLowerCase = _format.toLowerCase();
```
Comment (CordlessWool) [inline runtime/helpers/dateTimeHelpers.js:21]: I think in the linter rules it is not allowed to use _ before an variable. Maybe we have to include the linter roles to integration tests

File: runtime/helpers/APIhelpers.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+/**
+	*  API call for GET, PUT, POST and DELETE functionality
+	* @param url
+	* @param method
+	* @param body
+	* @param fileName
+	* @param statusCode
+	* @type {{ GET: receive all info, POST: create, PUT: edit / update, DELETE: remove info }},
+	*/
+	apiCall: function (url, method, body, fileName, statusCode) {
```
Comment (CordlessWool) [inline runtime/helpers/APIhelpers.js:14]: for what is this funktion? Integration tests should test the hole system (inlcuding frontend) and not only testing the api???

**Comment by CordlessWool** [comment]
please fill up the pull request template. There are not created to only to remove them ;)

File: runtime/helpers/APIhelpers.js (hunk 2 above)
Comment (kgorner) [reply runtime/helpers/APIhelpers.js:14]: This already was in Yulia's code. She might want to use it later on.

File: runtime/helpers/dateTimeHelpers.js (hunk 1 above)
Comment (kgorner) [reply runtime/helpers/dateTimeHelpers.js]: Corrected with the latest commit and push

File: runtime/helpers/APIhelpers.js (hunk 2 above)
Comment (yuliashikhareva) [reply runtime/helpers/APIhelpers.js:14]: this is just one function among many for concrete api test. That doesn't mean we use only such methods, but integrate them.

Comment (CordlessWool) [reply runtime/helpers/APIhelpers.js:14]: so it is not used and api tests should not done in this repo, right? So just remove it?

Comment (yuliashikhareva) [reply runtime/helpers/APIhelpers.js:14]: no please let it stay, we might need it in the future.

**Comment by kgorner** [comment]
23 scenarios (23 passed), file helpers.js is deleted, please review

File: page-objects/administration.js [hunk 3]
```diff
@@ -13,8 +13,9 @@ let newPassword = "Schulcloud1!"
 module.exports = {
 goToAdministration: function() {
     let url = Admin.urlAdministration;
-    return helpers.loadPage(url, 20);
+    return elementHelpers.loadPage(url, 10);
```
Comment (CordlessWool) [inline page-objects/administration.js:16]: is the number change wanted here?

Comment (kgorner) [reply page-objects/administration.js:16]: Das ist unabsichtlich passiert. Danke, dass du so genau geschaut hast! Ich habe es im nächsten Branch zurückgesetzt.

**Pull request merged**